apps/src/templates/sectionProgress/sectionProgressTestHelpers.js

```diff
@@ -150,14 +151,14 @@ function randomReviewState() {
 
 function getScriptData(numLessons) {
   return {
-    id: 162,
+    id: 1,
```

Comment:
Yes - The ID has to match what is in the unit selector dropdown and the fake things we seed the unit selector dropdown with defaults to 1 now

Reply:
Oh, I see.  I don't know how much work it would be, but it would be nice if the tests were more self-contained and the data was defined here so there's not coupling between different unit tests.

Reply:
This was the data I had to update for the Storybook for ProgressTableView so I'm not planning to work on cleaning it up as part of this work.